tests/mock_tables/dbconnector.py

```diff
@@ -86,6 +95,7 @@ def __init__(self, *args, **kwargs):
         topo = kwargs.pop('topo')
         namespace = kwargs.pop('namespace')
         db_name = kwargs.pop('db_name')
+        self.decode_responses = kwargs.pop('decode_responses', False) == True
```

Comment:
is `== True` necessary here?

Reply:
I asked this on Qi's earlier similar PR for swsssdk :)

It normalizes `decode_responses` to always be a Bool. E.g., if `kwargs.pop('decode_responses', False)` returns `None` or some other unexpected value.

Reply:
I see, so this is similar to `!!<bool>` trick in C. It appears to me that is is a bug in the downstream rather that some we should work around it??

Reply:
There is no bug in downstream. Here I just program conservatively.